docs_user/modules/proc_migrating-mon-and-mgr-from-controller-nodes.adoc

```diff
@@ -1,21 +1,21 @@
 [id="migrating-mon-and-mgr-from-controller-nodes_{context}"]
 
-= Migrating Ceph Monitor and Ceph Manager daemons to Red Hat Ceph Storage nodes
+= Migrating Ceph Monitor and Ceph Manager daemons to {Ceph} nodes
 //kgilliga: I'm trying to understand the purpose of this procedure. Is this procedure a prescriptive way for customers to migrate Ceph Monitor and Ceph manager daemons from controller nodes to Red Hat Ceph Storage nodes? Or are we recommending that customers create a proof of concept before doing the actual migration? And are oc0-controller-1 and oc0-ceph-0 just examples of the names of nodes for the purposes of this procedure? Note: The SME addressed these questions in the PR. This procedure needs more work. It should not be a POC.
```

Comment:
I think L4 should go away unless we want to take it in a follow up PR

Reply:
I removed most of the note, but I changed it to say that the procedure should not be a POC and needs to be revisited.